### PR TITLE
refactor: update structure

### PR DESCRIPTION
**Pull request by jleveugle**
* Remove iaas and paas abstract state
* Move all old project view to /legacy folder
* Update URLs to be more logic and friendly-use
* Add missing views due to new structure (list of projects, project view)

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/project.controller.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { actions, links } from './project.constants';
+
+export default class ProjectController {
+  /* @ngInject */
+  constructor($state, $stateParams, OvhApiCloudProject) {
+    this.$state = $state;
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/project.controller.js]:
```suggestion
  
```
doesn't seem to be used

File: packages/manager/modules/pci/src/projects/project/project.constants.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  },
+  {
+    translation: 'pci_projects_project_documentation_see_all_public_cloud_guides',
+    href: '#',
+  },
+];
+
+export default {
+  actions,
+  links,
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/project.constants.js]: Maybe those should be in CAPS as they are `constants`

File: packages/manager/modules/pci/src/projects/project/project.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            <oui-header-tabs-item text="Home" ui-sref="pci.projects.project({ serviceName: $ctrl.project.project_id })" active></oui-header-tabs-item>
+            <oui-header-tabs-item text="Parameters" ui-sref="pci.projects.project({ serviceName: $ctrl.project.project_id })" disabled></oui-header-tabs-item>
+        </oui-header-tabs>
+        <div class="py-5" data-ui-view>
+            <div class="container-fluid px-0">
+                <div class="row">
+                    <div class="col-12 col-sm-6 col-lg-4 col-xl mb-4 mb-xl-0" data-ng-repeat="action in $ctrl.actions track by $index" data-ng-class="{ 'pl-xl-0': $first, 'pr-xl-0': $last }">
+                        <a class="text-center w-100 d-block oui-tile py-4" href="#" title=":: action.translation | translate">
+                            <span class="oui-icon oui-icon-add fa-2x" aria-hidden="true"></span>
+                            <div class="mt-3 w-100 text-truncate" data-ng-bind=":: action.translation | translate"></div>
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/project.html]:
```suggestion
                            <div class="mt-3 w-100 text-truncate" data-translate=":: action.translation"></div>
```
?

File: packages/manager/modules/pci/src/projects/project/project.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+                        </a>
+                    </div>
+                </div>
+                <oui-tile class="mt-5" heading="{{ ::'pci_projects_project_documentation' | translate }}">
+                    <oui-tile-button external data-ng-repeat="link in $ctrl.links track by $index" href="link.href">{{ ::link.translation | translate }}</oui-tile-button>
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/project.html]:
```suggestion
                    <oui-tile-button data-external data-ng-repeat="link in $ctrl.links track by $index" href="link.href">{{ ::link.translation | translate }}</oui-tile-button>
```

File: packages/manager/modules/pci/src/projects/index.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-api-wrappers'; // should be a peer dependency of ovh-api-services
```
[the remark this answers is not in the document]
Comment (cbourgois) [reply packages/manager/modules/pci/src/projects/index.js]: But it's not imported in `ovh-api-services`so we have to import it.

**Pull request merged**